File: C/9_Palindrome_Number.c
bool isPalindrome(int x) {
    long int p = 0;
    if(x < 0)
        return false;
    long int aux = x;
    while(aux != 0)
    {
        p = p * 10 + aux % 10;
        aux = aux / 10;
    }
    if(x == p)
        return true;
    return false;
}

File: Python/9_Palindrome_Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        p = 0
        if x < 0:
            return False
        aux = x
        while aux != 0:
            p = p * 10 + aux % 10
            aux = aux // 10
        if x == p:
            return True
        return False
